internal/persistence/persistors/mysqlstore/store_organization_test.go

```diff
@@ -0,0 +1,496 @@
+
+				modelhelpers.AssertNonEmptyOrganizations(t, paginated.Organizations)
+			},
+		},
+		{
+			name: "success-filter-names-in",
+			filter: &model.OrganizationFilters{
+				OrganizationNameIn: []string{"Organization A", "Organization B"},
+			},
+			mutations: func(t *testing.T, db *sqlx.DB) {
+				entry := []mysqlmodel.Organization{
+					{ID: 1,
+						Name:          "Organization A",
+						CreatedBy:     null.IntFrom(2),
+						LastUpdatedBy: null.IntFrom(3),
+						CreatedAt:     time.Now(),
+						LastUpdatedAt: null.TimeFrom(time.Now()),
+						IsActive:      true}, {ID: 2,
+						Name:          "Organization B",
+						CreatedBy:     null.IntFrom(2),
+						LastUpdatedBy: null.IntFrom(3),
+						CreatedAt:     time.Now(),
+						LastUpdatedAt: null.TimeFrom(time.Now()),
+						IsActive:      true},
+				}
+
+				for _, org := range entry {
+					err := org.Insert(context.Background(), db, boil.Infer())
+					require.NoError(t, err, "error inserting sample data")
+				}
+			},
+			assertions: func(t *testing.T, db *sqlx.DB, paginated *model.PaginatedOrganizations, err error) {
+				require.NoError(t, err, "unexpected error")
+				require.NotNil(t, paginated, "unexpected nil paginated")
+				require.NotNil(t, paginated.Organizations, "unexpected nil organization")
+				require.NotNil(t, paginated.Pagination, "unexpected nil pagination")
+				assert.True(t, len(paginated.Organizations) == 2, "unexpected greater than 1 organization")
+				assert.True(t, paginated.Pagination.RowCount == 2, "unexpected count to be greater than 1 organization")
+
+				modelhelpers.AssertNonEmptyOrganizations(t, paginated.Organizations)
+			},
+		},
+		{
+			name: "success-multiple-filters",
+			filter: &model.OrganizationFilters{
+				IdsIn:              []int{1},
+				OrganizationNameIn: []string{"Organization A"},
+			},
+			mutations: func(t *testing.T, db *sqlx.DB) {
+				entry := mysqlmodel.Organization{
+					ID:            1,
+					Name:          "Organization A",
+					CreatedBy:     null.IntFrom(2),
+					LastUpdatedBy: null.IntFrom(3),
+					CreatedAt:     time.Now(),
+					LastUpdatedAt: null.TimeFrom(time.Now()),
+					IsActive:      true,
+				}
+				err := entry.Insert(context.Background(), db, boil.Infer())
+				require.NoError(t, err, "error inserting sample data")
+			},
+			assertions: func(t *testing.T, db *sqlx.DB, paginated *model.PaginatedOrganizations, err error) {
+				require.NoError(t, err, "unexpected error")
+				require.NotNil(t, paginated, "unexpected nil paginated")
+				require.NotNil(t, paginated.Organizations, "unexpected nil organization")
+				require.NotNil(t, paginated.Pagination, "unexpected nil pagination")
+				assert.True(t, len(paginated.Organizations) == 1, "unexpected greater than 1 organization")
+				assert.True(t, paginated.Pagination.RowCount == 1, "unexpected count to be greater than 1 organization")
+
+				modelhelpers.AssertNonEmptyOrganizations(t, paginated.Organizations)
+			},
```

Comment:
return the id in mutate so you don't have to deal with static numbers

Reply:
Sorry. We're a bit confused. Where exactly are the static number here? What we're doing is making the ID field in the entry dynamic now.